Project/Assets/Prairie/Framework/Script/Twine/TwineNode.cs

```diff
@@ -73,149 +79,157 @@ private void Start()
+                    optionButtonStyle.normal.textColor = Color.gray;
+
+                    selectedOptionIndex = GUILayout.SelectionGrid(selectedOptionIndex, this.childrenNames, 1, optionButtonStyle);
+                }
+                GUILayout.EndScrollView();
+                GUI.EndGroup ();
+
             }
+            else if (this.enabled && !this.isMinimized && !allMinimized) {
+                float frameWidth = Math.Min(Screen.width / 3, 350);
+                float frameHeight = Math.Min(Screen.height / 2, 500);
+                Rect frame = new Rect (10, 10, frameWidth, frameHeight);
+                GUI.BeginGroup (frame);
+                GUIStyle style = new GUIStyle (GUI.skin.box);
+                style.wordWrap = true;
+                style.fixedWidth = frameWidth;
+                GUILayout.Box (this.content, style);
+
+                if (isDecisionNode) {
+                    GUIStyle decisionHintStyle = new GUIStyle (style);
+                    decisionHintStyle.fontStyle = FontStyle.BoldAndItalic;
+
+                    if (!isOptionsGuiOpen) {
+                        GUILayout.Box ("Press TAB to progress in the story...", decisionHintStyle);
+                    } else {
+                        GUILayout.Box ("Press TAB to scroll, E to close, ENTER to choose", decisionHintStyle);
+                    }
+                }
 
-            GUI.EndGroup();
-        }
-        else if (fanfold) //Draw a bunch of boxes
-        {
-            float frameWidth = Math.Min(Screen.width / 3, 150);
-            float frameHeight = Math.Min(Screen.height / 2, 500);
-            int index = TwineNodeList.IndexOf(this);
-            Rect frame = new Rect(10 + index * 150, 10, frameWidth, frameHeight);
-            GUI.BeginGroup(frame);
-            GUIStyle style = new GUIStyle(GUI.skin.box);
-            style.wordWrap = true;
-            style.fixedWidth = frameWidth;
-            GUILayout.Box(this.content, style);
-
-            GUI.EndGroup();
+                if (this.isOptionsGuiOpen) {
+                    // Draw list of buttons for the possible children nodes to visit:
+                    GUIStyle optionButtonStyle = new GUIStyle (GUI.skin.button);
+                    optionButtonStyle.fontStyle = FontStyle.Italic;
+                    optionButtonStyle.wordWrap = true;
 
-        }
-        else if (this.enabled && !this.isMinimized) //Draw just this box
-        {
-            float frameWidth = Math.Min(Screen.width / 3, 350);
-            float frameHeight = Math.Min(Screen.height / 2, 500);
-            Rect frame = new Rect(10, 10, frameWidth, frameHeight);
-            GUI.BeginGroup(frame);
-            GUIStyle style = new GUIStyle(GUI.skin.box);
-            style.wordWrap = true;
-            style.fixedWidth = frameWidth;
-            GUILayout.Box(this.content, style);
+                    // Set highlighted button to have green text (this state is called `onNormal`):
```

Comment:
This is a good example to my last comment. You need to change 'green' to 'white' here as well, when you changed it on line 175.